homeassistant/components/pid_thermostat/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""The PID controller integration."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.pid_controller import async_setup as async_pid_setup
```

Comment:
Integrations are not allowed to consume others.

Import from another integration is thus not allowed.